src/test/java/io/jenkins/plugins/artifact_manager_s3/JCloudsArtifactManagerTest.java

```diff
@@ -123,8 +123,8 @@ public String getContainer() {
             return delegate.getContainer();
         }
         @Override
-        public boolean isDeleteBlobs() {
-            return deleteBlobs != null ? deleteBlobs : delegate.isDeleteBlobs();
+        public boolean isDeleteArtifacts() {
+            return deleteBlobs != null ? deleteBlobs : delegate.isDeleteArtifacts();
```

Comment:
To avoid confusion, could also rename this field and corresponding method parameters.